Sources/Build/BuildPlan/BuildPlan.swift

```diff
@@ -266,22 +267,24 @@ public class BuildPlan: SPMBuildCore.BuildPlan {
         destinationBuildParameters: BuildParameters,
         toolsBuildParameters: BuildParameters,
         graph: ModulesGraph,
+        pluginConfiguration: PluginConfiguration? = nil,
+        pluginTools: [ResolvedModule.ID: [String: PluginTool]] = [:],
         additionalFileRules: [FileRuleDescription] = [],
-        buildToolPluginInvocationResults: [ResolvedModule.ID: [BuildToolPluginInvocationResult]] = [:],
-        prebuildCommandResults: [ResolvedModule.ID: [PrebuildCommandResult]] = [:],
+        pkgConfigDirectories: [AbsolutePath] = [],
         disableSandbox: Bool = false,
         fileSystem: any FileSystem,
         observabilityScope: ObservabilityScope
     ) throws {
         self.destinationBuildParameters = destinationBuildParameters
         self.toolsBuildParameters = toolsBuildParameters
         self.graph = graph
-        self.buildToolPluginInvocationResults = buildToolPluginInvocationResults
-        self.prebuildCommandResults = prebuildCommandResults
         self.shouldDisableSandbox = disableSandbox
         self.fileSystem = fileSystem
         self.observabilityScope = observabilityScope.makeChildScope(description: "Build Plan")
 
+        var buildToolPluginInvocationResults: [ResolvedModule.ID: [BuildToolPluginInvocationResult]] = [:]
+        var prebuildCommandInvocationResults: [ResolvedModule.ID: [PrebuildCommandResult]] = [:]
```

Comment:
Let's stick to the naming used in the SE proposal: "command plugins".

```suggestion
        var commandPluginInvocationResults: [ResolvedModule.ID: [PrebuildCommandResult]] = [:]
```

Reply:
Renamed!